lib/clients/script_scanner_gui/tree_view/Data.py

```diff
@@ -276,31 +276,46 @@ def setData(self, column, value):
             self._value = value
 
 class ScanNode(Node):
-    
+
     columns = 8
-    
+
     def __init__(self, name, info, parent=None):
         super(ScanNode, self).__init__(name, parent)
         self._collection = parent.name()
         self.set_full_info(info)
-    
+
     def set_full_info(self, info):
         limit_info, scan_info = info
-        self._units = limit_info[0].units
-        self._min = limit_info[0][self._units]
-        self._max = limit_info[1][self._units]
-        self._scan_start = scan_info[0][self._units]
-        self._scan_stop = scan_info[1][self._units]
+        try:
+            self._units = limit_info[0].units

```

Comment:
for limit_info it might be nice to have attributes instead of an array, such as limit.min, and limit.max.